### PR TITLE
added the documentation for dellctl images command.

### PR DESCRIPTION
**Pull request by VamsiSiddu-7**
# Description
dellctl images is a new cmd which is introduced in the 1.5.1 release. This cmd helps us in listing the Container images needed by csi driver. 

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/582 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by rsedlock1958 — APPROVED** [review]
I have reviewed this. No edits are required.

**Review by ashleyvjoy — DISMISSED** [review]

File: content/docs/references/cli/_index.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ This document outlines all dellctl commands, their intended use, options that ca
 | [dellctl schedule get](#dellctl-schedule-get) | Get schedules |
 | [dellctl encryption rekey](#dellctl-encryption-rekey) | Rekey an encrypted volume |
 | [dellctl encryption rekey-status](#dellctl-encryption-rekey-status) | Get status of an encryption rekey operation |
+| [dellctl images](#dellctl-images) | Prints a Table of the container images Driver needs |
```
Comment (khareRajshree) [inline content/docs/references/cli/_index.md:33]: Hi @VamsiSiddu-7 can you please rephrase the sentence to something like this.., "List the container images used by driver."
Thanks.

File: content/docs/references/cli/_index.md [hunk 2]
```diff
@@ -782,3 +783,58 @@ Encryption rekey status with name of the rekey object
...
  INFO Status of rekey request myrekey = completed
 ```
+
+
+### dellctl images
+
+Prints a Table of the container images Driver needs
+
+**NOTE.**: dellctl images Currently supports csi-vxflexos driver only.
+
```
Comment (khareRajshree) [inline content/docs/references/cli/_index.md:793]: Similar sentence rephrasing is needed here as well "List the container images used by driver."
Thanks.

File: content/docs/references/cli/_index.md (hunk 1 above)
Comment (VamsiSiddu-7) [reply content/docs/references/cli/_index.md]: done

File: content/docs/references/cli/_index.md (hunk 2 above)
Comment (VamsiSiddu-7) [reply content/docs/references/cli/_index.md]: done

**Comment by VamsiSiddu-7** [comment]
@khareRajshree I have addressed your review comments. Can you pelase review it again. Thanks

**Review by khareRajshree — DISMISSED** [review]

**Review by rsedlock1958 — DISMISSED** [review]
I have3 re3viewed this. No edits are required.

File: content/docs/references/cli/_index.md [hunk 3]
```diff
@@ -782,3 +783,58 @@ Encryption rekey status with name of the rekey object
 # dellctl encryption rekey-status myrekey
  INFO Status of rekey request myrekey = completed
 ```
+
+
+### dellctl images
+
+List the container images needed by csi driver
+
+**NOTE.**: dellctl images Currently supports csi-vxflexos driver only.
```
Comment (bharathsreekanth) [inline content/docs/references/cli/_index.md:792]: Currently -> currently (first letter caps not needed)

**Review by bharathsreekanth — DISMISSED** [review]
Minor edit requested, approving.

File: content/docs/references/cli/_index.md (hunk 3 above)
Comment (VamsiSiddu-7) [reply content/docs/references/cli/_index.md]: done

**Review by rsedlock1958 — APPROVED** [review]
I have reviewed this. No grammatical edits are required

**Review by KshitijaKakde — APPROVED** [review]

**Review by khareRajshree — APPROVED** [review]

**Pull request merged**